### ISSUE TITLE
mobile/ visible focus indicators on all interactive elements

### ISSUE THREAD

**Issue by serdarbahar** [bounswe/bounswe2025group5#565]
### Task Description
Visible focus indicators on all interactive elements should be present according to the accessibility requirements. This issue will deal with implementing the feature in the mobile application. The relevant [requirement](https://github.com/bounswe/bounswe2025group5/wiki/Requirements-of-CMPE451) is 2.1.2.8.

### Why Is This Needed?
- To follow accessibility guidelines throughout our project

### Acceptance Criteria
- [ ] visible focus indicators
 - indicators for all interactive elements is provided for all interactive elements, if applicable
 - if visible focus indicators are not applicable for mobile, the reason why

### Deadline
20.11.2025 - 23:59

**Comment by serdarbahar** [comment]
As noted in the [last meeting](https://github.com/bounswe/bounswe2025group5/wiki/Meeting-11-of-451), the feature is not applicable for the mobile application.
